presto-main/src/main/java/com/facebook/presto/operator/ArrayPositionLinks.java

```diff
@@ -14,9 +14,13 @@
 package com.facebook.presto.operator;
 
 import com.facebook.presto.spi.Page;
+import com.google.common.hash.Hasher;
```

Comment:
THe `This is is helpful to ensure that unspilled LookupSource is the same as
the one before spill.` commit message looks weird. Just write where it is used and why.